src/actions/transformCaseAction.ts

```diff
@@ -0,0 +1,77 @@
+module Plywood {
+  export type CaseType = 'upperCase' | 'lowerCase';
+  export class TransformCaseAction extends Action {
+    static UPPER = 'upperCase';
+    static LOWER = 'lowerCase';
```

Comment:
I think it is nice for these to match, so ether have it `static LOWER_CASE = 'lowerCase'` or `static LOWER = 'lower'` I am fine with ether